public/js/components/App.js

```diff
@@ -5,6 +5,8 @@ class App extends React.Component {
     this.state = {
       tiles: null,
       bounds: null,
+      geoJson: null,
+      useTiles: false,
```

Comment:
This line in state is where we can switch from using geojson, to using tiles. I figure having this there will make it easier for whichever one of us lays down a switch in the UI.

Reply:
👍